R/pam-tidiers.R

```diff
@@ -0,0 +1,90 @@
+' Tidying methods for pam objects
+#' 
+#' These methods summarize the results of Partitioning Around Medoids (PAM) clustering into three
+#' tidy forms. `tidy` describes the medoid and size of each cluster,
+#' `augment` adds the cluster assignments to the original data, and
+#' `glance` summarizes the average silhouette with of the clustering.
+#'
+#' @templateVar class pam
+#' @template title_desc_tidy
+#' 
+#' @param x An `pam` object returned from [cluster::pam()]
+#' @param col.names Dimension names. Defaults to the names of the variables in x.  Set to NULL to get names `x1, x2, ...`.
+#' @template param_unused_dots
+#' 
+#' @evalRd return_tidy(
+#'   size = "The size of each cluster",
+#'   max_diss ="The maximal dissimilarity between the observations in the cluster and the cluster's medoid",
+#'   av_diss= "The average dissimilarity between the observations in the cluster and the cluster's medoid", 
+#'   diameter="The diameter of the cluster",
```

Comment:
Would you mind doing spaces around the `=` to match tidyverse style?

Reply:
In general, documentation should be in grammatically correct sentences with complete punctuation (i.e. periods).